### PR TITLE
Dh10

### PR DESCRIPTION
**Pull request by freemangordon**

### CODE REVIEW

**Review by MerlijnWajer — APPROVED** [review]
Looks good to me!

File: configure.ac [hunk 1]
```diff
@@ -7,24 +7,17 @@
 # Released under the terms of LGPL version 2.1
 #
 
-AC_INIT(Makefile.am)
-VERSION=2.0.4
+AC_INIT([hildon-input-method], [2.0.4])
```
Comment (spinal84) [inline configure.ac:10]: Can be changed to
`AC_INIT([hildon-input-method], [2.1.53])`

File: debian/control [hunk 2]
```diff
@@ -1,20 +1,22 @@
 Source: hildon-input-method
 Section: x11
 Priority: optional
-Maintainer: Mohammad Anwari <Mohammad.Anwari@nokia.com>
-Build-Depends: debhelper, cdbs, pkg-config, libgtk2.0-dev (>= 2.14.7-1maemo15), libosso-dev (>=0.9.10-2), libgconf2-dev, libhildon1-dev, hildon-input-method-framework-dev (>= 1:2.1.66-1), libdbus-1-dev, maemo-launcher-dev (>= 0.23-1), gtk-doc-tools
+Maintainer: Ivaylo Dimitrov <ivo.g.dimitrov.75@gmail.com>
```
Comment (spinal84) [inline debian/control:4]: Changing maintainer is important thing - it's better to be done in a separate commit

**Comment by spinal84** [comment]
- Changing maintainer is important thing - it's better to be done in a separate commit
- Code looks good, commit messages look a bit sloppy. Please, use google translate.
"remove files that do<s>**es**</s> not belong to git"
"all the plugins will need to be rebuil**t**"

File: debian/control (hunk 2 above)
Comment (freemangordon) [reply debian/control:4]: Sure, if I was posting on LKML, which I am not.

**Comment by freemangordon** [comment]
>     * Changing maintainer is important thing - it's better to be done in a separate commit
> 
>     * Code looks good, commit messages look a bit sloppy. Please, use google translate.
>       "remove files that do**es** not belong to git"
>       "all the plugins will need to be rebuil**t**"

"Use google translate" is not really a good advice, google translate fails miserably when asked to translate anything more complicated than "здравей". Not to say it is obvious that those are simple typos. 

Not that my English is the best around, still, it is better than google translate.

And I would like to stress again that the tone of your comments sound impolite to say the least. I don't know (and honestly don't care) if it is the language barrier or something else, however, please correct that.

File: debian/control (hunk 2 above)
Comment (spinal84) [reply debian/control:4]: Don't you take Maemo Leste seriously?

**Comment by spinal84** [comment]
> And I would like to stress again that the tone of your comments sound impolite to say the least. I don't know (and honestly don't care) if it is the language barrier or something else, however, please correct that.

I am writing a case, this should be enough. No need to teach me how to communicate with people. 
We are not at the bar.

File: debian/control (hunk 2 above)
Comment (freemangordon) [reply debian/control:4]: are you kidding me?

**Comment by freemangordon** [comment]
> > And I would like to stress again that the tone of your comments sound impolite to say the least. I don't know (and honestly don't care) if it is the language barrier or something else, however, please correct that.
> 
> I am writing a case, this should be enough. No need to teach me how to communicate with people.
> We are not at the bar.

I am not going to teach you (seems you already know everything), I am just going to ignore your posts/comments from now on.

File: debian/control (hunk 2 above)
Comment (spinal84) [reply debian/control:4]: I think if the things go well, more developers will join us. In such a case it's better to show how it should be done properly from start. Instead of talking about LKML > Maemo-Leste, who cares?
